src/main/java/touro/chess/Board.java

```diff
@@ -1,9 +1,56 @@
 package touro.chess;
 
+import java.util.List;
+
 public class Board {
 
     private static final int COLUMNS = 8;
     private static final int ROWS = 8;
+    private Square[][] squares = new Square[COLUMNS][ROWS];
+
+    public Board()
+    {
+        for(int column = 0; column < COLUMNS; column++)
```

Comment:
Probably is a little easier to read if you set up all the Squares on a blank Board and then place the Pieces. 

Also you should have a test for this.

Reply:
I changed it back to that, but that makes the code less efficient because it is looping through the arrays twice. Is it worthwhile to sacrifice efficiency for readability in this case?

Reply:
You can do the "second time" without the loop.